internal-packages/workflow-designer-ui/src/editor/properties-panel/image-generation-node-properties-panel/input-panel.tsx

```
@@ -515,12 +478,12 @@ export function InputPanel({
 											`Expected file node, got ${source.node.content.type}`,
 										);
 									}
+									const pluralize = (word: string, count: number) =>
+										count === 1 ? word : `${word}s`;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)